### PR TITLE
Timepicker, list and tabs compos

### PR DESCRIPTION
**Pull request by virginieky**
- Timepicker
- List
- Tabs

### CODE REVIEW

File: buffet/src/assets/styles/colors.js [hunk 1]
```diff
@@ -14,11 +14,25 @@ const colors = {
...
+  greyHeader: '#F3F3F4',
+  greySubtitle: '#787E8F',
+  greyHover: '#F7F8F8',
+
+  // Specific to timepicker
+  blueIconTxt: '#367CF7',
+  blueIconBkgd: '#E8F0FA',
+
+  // Specific to tabs
+  greyTabBkgd: 'rgb(242, 242, 242)',
```
Comment (skigun) [inline buffet/src/assets/styles/colors.js]: Why not using `#F2F2F2` like other colors ?

File: buffet/src/combined/List/tests/List.test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// import React from 'react';
+// import { mount } from 'enzyme';
+// import { enzymeFind } from 'styled-components/test-utils';
+
+// import List from '../index';
```
Comment (skigun) [inline buffet/src/combined/List/tests/List.test.js]: 💡 You should never commit commented code

File: buffet/src/commonPropTypes/list/defaultProps.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export default {
+  list: [],
```
[the remark this answers is not in the document]
Comment (virginieky) [reply buffet/src/commonPropTypes/list/defaultProps.js]: Yes, maybe it should be named differently like rows

File: buffet/src/combined/List/tests/List.test.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+describe('<List />', () => {
+  it('Expect to have unit tests specified', () => {
```
[the remark this answers is not in the document]
Comment (virginieky) [reply buffet/src/combined/List/tests/List.test.js]: agreed

**Pull request merged**